meta-mender-core/recipes-mender/mender-client/mender-client_git.inc

```diff
@@ -17,8 +17,9 @@ def mender_autorev_if_git_version(d):
         return "f6ffa190892202263fdb75975059fbb201adab6a"
 SRCREV ?= '${@mender_autorev_if_git_version(d)}'
 
-# Force the version of master packages to compare very highly to other versions.
-PKGV = "${@'999+master+git${SRCPV}' if '${PV}'.startswith('master') else '${PV}'}"
+# Force the version of master and feature-branch packages to compare very highly
+# to other versions.
+PKGV = "${@'999+${PV}' if any(['${PV}'.startswith(p) for p in ['master', 'feature']]) else '${PV}'}"
```

Comment:
Nitpick. I would be a tiny bit more restrictive:
```suggestion
PKGV = "${@'999+${PV}' if any(['${PV}'.startswith(p) for p in ['master', 'feature-']]) else '${PV}'}"
```

Reply:
Good point, will fix!